projects/clinician-insights/daily-feedback.py

```diff
@@ -0,0 +1,413 @@
+    else:
+        statDF["GTE2hoursBetween54to70"] = np.nan
+
+    # 3.  >= 15 minutes < 54 mg/dl"
+    if statDF.loc[0, "totalBelow54"] > (15 / 5):
+        statDF["GTE15minBelow54"] = "FLAG (" + str(round(statDF.loc[0, "totalBelow54"] * 5)) + "min)"
+    else:
+        statDF["GTE15minBelow54"] = np.nan
+
+    return statDF
+
+
+def sort_and_pretty_stat_output(df):
+
+    for col in list(df):
+        if (("percent" in col) | ("cov" in col)):
+            df[col] = round(df[col] * 100, 1)
+
+    for col in ["mean_mgdL", "std_mgdL"]:
+        df[col] = round(df[col], 1)
+
+    df = df[["studyID",
+             "incompleteDataset",
+             "GTE4hoursNoCgmSignal",
+             "GTE2hoursBetween54to70",
+             "GTE15minBelow54",
+             "totalNumberCBGValues",
+             "totalNumberPossibleCBGvalues",
+             "startTime",
+             "endTime",
+             "percentOfExpectedData",
+             "mean_mgdL",
+             "std_mgdL",
+             "cov_mgdL",
+             "min_mgdL",
+             "median_mgdL",
+             "max_mgdL",
+             "percentBelow54",
+             "percentBelow70",
+             "percent70to140",
+             "percent70to180",
+             "percentAbove180",
+             "percentAbove250",
+             "totalBelow54",
+             "totalBelow70",
+             "total54to70",
+             "total70to140",
+             "total70to180",
+             "total180to250",
+             "totalAbove180",
+             "totalAbove250"]]
+
+    return df
+
+
+def get_timeZoneOffset(currentDate, userTz):
+    tz = timezone(userTz)
+    tzoNum = int(tz.localize(pd.to_datetime(currentDate) + timedelta(days=1)).strftime("%z"))
+    tzoHours = np.floor(tzoNum / 100)
+    tzoMinutes = round((tzoNum / 100 - tzoHours) * 100, 0)
+    tzoSign = np.sign(tzoHours)
+    tzo = int((tzoHours * 60) + (tzoMinutes * tzoSign))
+    return tzo
+
+
+def get_donor_lists(email, password, outputDonorList):
+    p = sub.Popen(["getusers", email,
+                   "-p", password, "-o",
+                   outputDonorList, "-v"], stdout=sub.PIPE, stderr=sub.PIPE)
+
+    output, errors = p.communicate()
+    output = output.decode("utf-8")
+    errors = errors.decode("utf-8")
+
+    if output.startswith("Successful login.\nSuccessful") is False:
+        sys.exit("ERROR with" + email +
+                 " ouput: " + output +
+                 " errorMessage: " + errors)
+
+    return
+
+
+def load_donors(outputDonorList):
+    donorList = []
+    if os.stat(outputDonorList).st_size > 0:
+        donorList = pd.read_csv(outputDonorList,
+                                header=None,
+                                usecols=[0, 1],
+                                names=["userID", "name"],
+                                low_memory=False)
+
+    return donorList
+
+
+def get_json_data(email, password, userid, outputFilePathName, startDate, endDate):
```

Comment:
Seems like this should go to `tidals`...

Reply:
yes, same comment as below